build/README.md

```diff
@@ -0,0 +1,34 @@
+# Everoute CNI QuickStart
+
+# Backgroud
+Everoute CNI provides standard K8S CNI function, 
+including network support for POD communication,
+and standard NetworkPolicy network strategy (currently supports up to k8s v1.21)
+
+## Image
+### build image
+```shell
+git glone https://github.com/everoute/everoute.git
+cd everoute
+make images
+```
+images need to be manually distributed to each node.
+
+### public docker hub
+hub.docker.io (Pending)
+
+### private docker hub
+harbor.smartx.com - everoute/release
+
+## Deployment
+```shell
+wget https://raw.githubusercontent.com/everoute/everoute/main/deploy/everoute.yaml
+kubectl apply -f everoute.yaml
+```
+
+## Check
+By default everoute has one controller and one agent on each node
```

Comment:
Why each node need to has one controller?

Reply:
Sry, I mean one controller in whole cluster and one agent on each node